addons/mail/static/src/components/message/message.xml

```diff
@@ -157,7 +158,17 @@
                         </t>
                     </t>
                     <div class="o_Message_content" t-ref="content">
-                        <div class="o_Message_prettyBody" t-ref="prettyBody"/><!-- message.prettyBody is inserted here from _update() -->
+                        <t t-if="message.hasSendError">
+                            <span class="not-delivred">
+                                <i class="fa fa-exclamation-circle" />
+                                This message was not delivered
+                            </span>
+                            <br />
```

Comment:
Maybe consider flex column on parent instead of `<br>` for style purposes. I don't think `<br>` should ever be used outside of specific text that needs to be formatted in a certain way (eg. a poem).